src/odemis/driver/actuator.py

```diff
@@ -1417,23 +1421,33 @@ def moveAbs(self, pos):
         return f
 
     def _doMoveAbs(self, pos):
-        axis, distance = list(pos.items())[0]
-        logging.debug("Moving axis %s (-> %s) to %g", self._axis, self._caxis, distance)
+        axis, req_pos = list(pos.items())[0]
+
+        # if the actuator is already at the requested position skip movement
+        # this to prevent the actuator to move due to difference dep pos and requested pos
+        # and en up in the same place again. For instance, this is typical when anti-backlash is used.
```

Comment:
This comment needs a little bit of love. I'd write it like:
> If the actuator is already at the requested position, skip the move.
> This prevents the actuator from moving just due to tiny difference between actual pos and requested pos,
> in which case it typically ends up in the same place again. For instance, this can happen when anti-backlash is used.